tests/expectations/tests/layout.rs

```diff
@@ -38,7 +38,7 @@ impl <T> ::std::clone::Clone for __IncompleteArrayField<T> {
 }
 impl <T> ::std::marker::Copy for __IncompleteArrayField<T> { }
 #[repr(C, packed)]
-#[derive(Debug, Default, Copy)]
```

Comment:
Then this was a pretty egregious bug; pretty sad we didn't catch that :(

Definitely shouldn't be deriving `Copy` for incomplete arrays, as we have no idea what the correct thing to do in the scenario is...